test/e2e/framework/docker.go

```diff
@@ -55,6 +55,7 @@ func (d *Docker) GetLog() (string, string) {
 	var stdout, stderr bytes.Buffer
 
 	// get stdout and stderr of `docker log {d.Name}` command
+	// #nosec G204 -- the caller-controlled value is only used as the logs argument
```

Comment:
If anyone else is wondering, here's a very good reference for what's vulnerable with `os/exec.Command` and what's not:

https://semgrep.dev/docs/cheat-sheets/go-command-injection/#1a-running-os-commands-with-execcommand

This usage isn't one of the vulnerable patterns, and we meet all of the mitigation suggestions.